covidWallet/ios/Podfile

```diff
@@ -44,6 +44,15 @@ def flipper_post_install(installer)
   end
 end
 
+target 'ReactNativeBarcode' do
+  # Permissions
+  permissions_path = '../node_modules/react-native-permissions/ios'
+  pod 'Permission-Camera', :path => "#{permissions_path}/Camera.podspec"
+
+  # Pods for ReactNativeBarcode
+  ...
+end
+
```

Comment:
Please do not add any iOS changes if you cannot test them before making PR. This code snippet has syntax error and logical error. Please remove it, if you want to add iOS integration you need to add following line after line 57:

`pod 'Permission-Camera', :path => "../node_modules/react-native-permissions/ios/Camera.podspec"`

 Like this 
```
target 'covidWallet' do
  # Pods for covidWallet
  pod 'Permission-Camera', :path => "../node_modules/react-native-permissions/ios/Camera.podspec"
```